projects/core/components/textfield/textfield.template.html

```diff
@@ -26,10 +26,10 @@
     <ng-content select="tui-icon" />
 </span>
 <span
-    *ngIf="control?.value != null"
+    *ngIf="control?.value != null || cva?.value() != null"
```

Comment:
Will it fix issue only for our custom controls `class Tui*** extends TuiControl` ?
What's is about basic `Textfield` usage ?
https://taiga-ui.dev/components/textfield#basic

Should we use `toSignal` + `control.valueChanges` instead ?

Reply:
Check this case
https://github.com/taiga-family/taiga-ui/issues/11324#issuecomment-3077643102

It is still reproducible even with your new changes